orchagent/muxorch.cpp

```diff
@@ -790,9 +790,9 @@ void MuxAclHandler::createMuxAclTable(sai_object_id_t port, string strTable)
 
     acl_table.type = ACL_TABLE_DROP;
     acl_table.id = strTable;
-    acl_table.link(port);
     acl_table.stage = ACL_STAGE_INGRESS;
     gAclOrch->addAclTable(acl_table);
+    bindAllPorts(acl_table);
```

Comment:
As you can see from line 783, the same table shall be used if pfcwd has already created this table. You may want to apply the same fix [here ](https://github.com/Azure/sonic-swss/blob/master/orchagent/pfcactionhandler.cpp#L410) as well -

Reply:
sure. I can update the PR with the changes but won't be able to verify as Cisco platform does not use drop ACL table for PFC wd. Is it fine to change?

Reply:
Sure. I can update the PR with requested changes. But cisco platform does not use drop ACL for PFC wd so I won't be able to verify it. Is it fine to change?

Reply:
ok, then leave it for now and revisit later